docs/EnvironmentVariables.md

```diff
@@ -21,3 +21,14 @@
 * `ZM_BACKEND_RPCAPI_LOGJSON`: Setting it to any thruthy value (non-empty
   string or non-zero number) will configure the logger to log in JSON format,
   undefined by default.
+
+## Variables specific to testing
+
+* `TARGET`: Set the database to use. If undefined, will assume SQLite.
```

Comment:
If think the following is more correct and better language:
```
If empty or undefind, "SQLite" will be assumed.
```

Reply:
updated